# Swedish message translation file for plpython
# Copyright (C) 2017 PostgreSQL Global Development Group
# This file is distributed under the same license as the PostgreSQL package.
# Dennis Björklund <db@zigo.dhs.org>, 2017, 2018, 2019, 2020.
#
msgid ""
msgstr ""
"Project-Id-Version: PostgreSQL 9.5\n"
"Report-Msgid-Bugs-To: pgsql-bugs@postgresql.org\n"
"POT-Creation-Date: 2020-08-30 20:54+0000\n"
"PO-Revision-Date: 2020-08-31 07:20+0200\n"
"Last-Translator: Dennis Björklund <db@zigo.dhs.org>\n"
"Language-Team: Swedish <sv@li.org>\n"
"Language: sv\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"

#: plpy_cursorobject.c:98
#, c-format
msgid "plpy.cursor expected a query or a plan"
msgstr "plpy.cursor förväntade sig en fråga eller en plan"

#: plpy_cursorobject.c:171
#, c-format
msgid "plpy.cursor takes a sequence as its second argument"
msgstr "plpy.cursor tar en sekvens som sitt andra argument"

#: plpy_cursorobject.c:187 plpy_spi.c:217
#, c-format
msgid "could not execute plan"
msgstr "kunde inte exekvera plan"

#: plpy_cursorobject.c:190 plpy_spi.c:220
#, c-format
msgid "Expected sequence of %d argument, got %d: %s"
msgid_plural "Expected sequence of %d arguments, got %d: %s"
msgstr[0] "Förväntade sekvens med %d argument, fick %d: %s"
msgstr[1] "Förväntade sekvens med %d argument, fick %d: %s"

#: plpy_cursorobject.c:340
#, c-format
msgid "iterating a closed cursor"
msgstr "itererar med en stängd markör"

#: plpy_cursorobject.c:348 plpy_cursorobject.c:413
#, c-format
msgid "iterating a cursor in an aborted subtransaction"
msgstr "itererar med en markör i en avbruten subtransaktion"

#: plpy_cursorobject.c:405
#, c-format
msgid "fetch from a closed cursor"
msgstr "hämta från en stängd markör"

#: plpy_cursorobject.c:482
#, c-format
msgid "closing a cursor in an aborted subtransaction"
msgstr "stänger en markör i en avbruten subtransaktion"

#: plpy_elog.c:108 plpy_elog.c:109 plpy_plpymodule.c:440
#, c-format
msgid "%s"
msgstr "%s"

#: plpy_exec.c:91
#, c-format
msgid "unsupported set function return mode"
msgstr "ej supportat returläge för mängdfunktion"

#: plpy_exec.c:92
#, c-format
msgid "PL/Python set-returning functions only support returning one value per call."
msgstr "PL/Python mängdreturnerande funktioner stöder bara ett värde per anrop."

#: plpy_exec.c:104
#, c-format
msgid "returned object cannot be iterated"
msgstr "returnerat objekt kan inte itereras"

#: plpy_exec.c:105
#, c-format
msgid "PL/Python set-returning functions must return an iterable object."
msgstr "PL/Python mängdreturnerande funktioner måste returnera ett itererbart objekt."

#: plpy_exec.c:130
#, c-format
msgid "error fetching next item from iterator"
msgstr "fel vid hämtning av nästa del från iteratorn"

#: plpy_exec.c:165
#, c-format
msgid "PL/Python function with return type \"void\" did not return None"
msgstr "PL/Python-funktion med returtyp \"void\" returnerade inte None"

#: plpy_exec.c:291 plpy_exec.c:317
#, c-format
msgid "unexpected return value from trigger procedure"
msgstr "oväntat returvärde från utlösarprocedur"

#: plpy_exec.c:292
#, c-format
msgid "Expected None or a string."
msgstr "Förväntade None eller en sträng."

#: plpy_exec.c:307
#, c-format
msgid "PL/Python trigger function returned \"MODIFY\" in a DELETE trigger -- ignored"
msgstr "PL/Python-utlösarfunktion returnerade \"MODIFY\" i en DELETE-utlösare -- ignorerad"

#: plpy_exec.c:318
#, c-format
msgid "Expected None, \"OK\", \"SKIP\", or \"MODIFY\"."
msgstr "Förväntade None, \"OK\", \"SKIP\" eller \"MODIFY\"."

#: plpy_exec.c:399
#, c-format
msgid "PyList_SetItem() failed, while setting up arguments"
msgstr "PyList_SetItem() misslyckades vid uppsättning av argument"

#: plpy_exec.c:403
#, c-format
msgid "PyDict_SetItemString() failed, while setting up arguments"
msgstr "PyDict_SetItemString() misslyckades vid uppsättning av argument"

#: plpy_exec.c:415
#, c-format
msgid "function returning record called in context that cannot accept type record"
msgstr "en funktion med post som värde anropades i sammanhang där poster inte kan godtagas."

#: plpy_exec.c:453
#, c-format
msgid "while creating return value"
msgstr "vid skapande av returvärde"

#: plpy_exec.c:477
#, c-format
msgid "could not create new dictionary while building trigger arguments"
msgstr "kunde inte skapa ny katalog vid byggande av utlösarargument"

#: plpy_exec.c:666
#, c-format
msgid "TD[\"new\"] deleted, cannot modify row"
msgstr "TD[\"new\"] raderad, kan inte modifiera rad"

#: plpy_exec.c:671
#, c-format
msgid "TD[\"new\"] is not a dictionary"
msgstr "TD[\"new\"] är inte en dictionary"

#: plpy_exec.c:696
#, c-format
msgid "TD[\"new\"] dictionary key at ordinal position %d is not a string"
msgstr "TD[\"new\"] dictionary-nyckel vid numerisk position %d är inte en sträng"

#: plpy_exec.c:703
#, c-format
msgid "key \"%s\" found in TD[\"new\"] does not exist as a column in the triggering row"
msgstr "nyckel \"%s\" hittad i TD[\"new\"] finns inte som en kolumn i den utlösande raden"

#: plpy_exec.c:783
#, c-format
msgid "while modifying trigger row"
msgstr "vid modifiering av utlösande rad"

#: plpy_exec.c:844
#, c-format
msgid "forcibly aborting a subtransaction that has not been exited"
msgstr "tvingar avbrytande av subtransaktion som inte har avslutats"

#: plpy_main.c:124
#, c-format
msgid "Python major version mismatch in session"
msgstr "Pythons major-version matchar inte i sessionen"

#: plpy_main.c:125
#, c-format
msgid "This session has previously used Python major version %d, and it is now attempting to use Python major version %d."
msgstr "Sessionen har tidigare använt Python med major-version %d och nu försöker den använda Python med major-version %d."

#: plpy_main.c:127
#, c-format
msgid "Start a new session to use a different Python major version."
msgstr "Starta en ny session för att använda en Python med annan major-version."

#: plpy_main.c:150
#, c-format
msgid "multiple Python libraries are present in session"
msgstr "multipla Python-bibliotek är aktiva i sessionen"

#: plpy_main.c:151
#, c-format
msgid "Only one Python major version can be used in one session."
msgstr "Bara en major-version av Python kan användas i en session."

#: plpy_main.c:167
#, c-format
msgid "untrapped error in initialization"
msgstr "ej fångar fel i initiering"

#: plpy_main.c:190
#, c-format
msgid "could not import \"__main__\" module"
msgstr "kunde inte importera \"__main__\"-modul"

#: plpy_main.c:195
#, c-format
msgid "could not create globals"
msgstr "kundew inte skapa globaler"

#: plpy_main.c:199
#, c-format
msgid "could not initialize globals"
msgstr "kunde inte initierar globaler"

#: plpy_main.c:409
#, c-format
msgid "PL/Python function \"%s\""
msgstr "PL/Python-funktion \"%s\""

#: plpy_main.c:416
#, c-format
msgid "PL/Python anonymous code block"
msgstr "PL/Python anonymt kodblock"

#: plpy_planobject.c:126
#, c-format
msgid "plan.status takes no arguments"
msgstr "plan.status tar inga argument"

#: plpy_plpymodule.c:181 plpy_plpymodule.c:184
#, c-format
msgid "could not import \"plpy\" module"
msgstr "kunde inte importera \"plpy\"-modul"

#: plpy_plpymodule.c:199
#, c-format
msgid "could not create the spiexceptions module"
msgstr "kunde inte skapa modulen spiexceptions"

#: plpy_plpymodule.c:207
#, c-format
msgid "could not add the spiexceptions module"
msgstr "kunde inte lägga till modulen spiexceptions"

#: plpy_plpymodule.c:236
#, c-format
msgid "could not create exception \"%s\""
msgstr "kunde inte skapa undantag \"%s\""

#: plpy_plpymodule.c:271 plpy_plpymodule.c:275
#, c-format
msgid "could not generate SPI exceptions"
msgstr "kunde inte skapa SPI-undantag"

#: plpy_plpymodule.c:408
#, c-format
msgid "could not unpack arguments in plpy.elog"
msgstr "kunde inte packa upp argument i plpy.elog"

#: plpy_plpymodule.c:416
msgid "could not parse error message in plpy.elog"
msgstr "kunde inte parsa felmeddelande i plpy.elog"

#: plpy_procedure.c:221
#, c-format
msgid "trigger functions can only be called as triggers"
msgstr "Triggningsfunktioner kan bara anropas vid triggning."

#: plpy_procedure.c:226
#, c-format
msgid "PL/Python functions cannot return type %s"
msgstr "PL/Python-funktioner kan inte returnera typ %s"

#: plpy_procedure.c:308
#, c-format
msgid "PL/Python functions cannot accept type %s"
msgstr "PL/Python-funktioner kan inte ta emot typ %s"

#: plpy_procedure.c:406
#, c-format
msgid "could not compile PL/Python function \"%s\""
msgstr "kunde inte kompilera PL/Python-funktion \"%s\""

#: plpy_procedure.c:409
#, c-format
msgid "could not compile anonymous PL/Python code block"
msgstr "kunde inte kompilera anonymt PL/Python-kodblock"

#: plpy_resultobject.c:145 plpy_resultobject.c:165 plpy_resultobject.c:185
#, c-format
msgid "command did not produce a result set"
msgstr "kommandot producerade inte en resultatmängd"

#: plpy_spi.c:57
#, c-format
msgid "second argument of plpy.prepare must be a sequence"
msgstr "andra argumentet till plpy.prepare måste vara en sekvens"

#: plpy_spi.c:106
#, c-format
msgid "plpy.prepare: type name at ordinal position %d is not a string"
msgstr "plpy.prepare: typnamn vid numerisk position %d är inte en sträng"

#: plpy_spi.c:182
#, c-format
msgid "plpy.execute expected a query or a plan"
msgstr "plpy.execute förväntade en fråga eller en plan"

#: plpy_spi.c:201
#, c-format
msgid "plpy.execute takes a sequence as its second argument"
msgstr "plpy.execute tar en sekvens som sitt andra argument"

#: plpy_spi.c:325
#, c-format
msgid "SPI_execute_plan failed: %s"
msgstr "SPI_execute_plan misslyckades: %s"

#: plpy_spi.c:367
#, c-format
msgid "SPI_execute failed: %s"
msgstr "SPI_execute misslyckades: %s"

#: plpy_subxactobject.c:122
#, c-format
msgid "this subtransaction has already been entered"
msgstr "denna subtransaktion har redan gåtts in i"

#: plpy_subxactobject.c:128 plpy_subxactobject.c:180
#, c-format
msgid "this subtransaction has already been exited"
msgstr "denna subtransaktion har redan avslutat"

#: plpy_subxactobject.c:174
#, c-format
msgid "this subtransaction has not been entered"
msgstr "denna subtransaktion har inte gåtts in i"

#: plpy_subxactobject.c:186
#, c-format
msgid "there is no subtransaction to exit from"
msgstr "det finns ingen subtransaktion att avsluta från"

#: plpy_typeio.c:302
#, c-format
msgid "could not create new dictionary"
msgstr "kunde inte skapa ny katalog"

#: plpy_typeio.c:564
#, c-format
msgid "could not import a module for Decimal constructor"
msgstr "kunde inte importera en modul för Decimal-konstruktorn"

#: plpy_typeio.c:568
#, c-format
msgid "no Decimal attribute in module"
msgstr "inga Decimal-attribut i modulen"

#: plpy_typeio.c:574
#, c-format
msgid "conversion from numeric to Decimal failed"
msgstr "konvertering från numeric till Decimal misslyckades"

#: plpy_typeio.c:649
#, c-format
msgid "cannot convert multidimensional array to Python list"
msgstr "kan inte konvertera multidimensionell array till Python-lista"

#: plpy_typeio.c:650
#, c-format
msgid "PL/Python only supports one-dimensional arrays."
msgstr "PL/Python stöder bara endimensionella array:er."

#: plpy_typeio.c:656
#, c-format
msgid "could not create new Python list"
msgstr "kunde inte skapa ny Python-lista"

#: plpy_typeio.c:715
#, c-format
msgid "could not create bytes representation of Python object"
msgstr "kunde inte skapa byte-representation av Python-objekt"

#: plpy_typeio.c:820
#, c-format
msgid "could not create string representation of Python object"
msgstr "kunde inte skapa strängrepresentation av Python-objekt"

#: plpy_typeio.c:831
#, c-format
msgid "could not convert Python object into cstring: Python string representation appears to contain null bytes"
msgstr "kunde inte konvertera Python-objekt till cstring: Python-strängrepresentationen verkar innehålla noll-bytes"

#: plpy_typeio.c:877
#, c-format
msgid "return value of function with array return type is not a Python sequence"
msgstr "returvärde för funktion med array-returtyp är inte en Python-sekvens"

#: plpy_typeio.c:992
#, c-format
msgid "key \"%s\" not found in mapping"
msgstr "nyckeln \"%s\" hittades inte i mapping"

#: plpy_typeio.c:993
#, c-format
msgid "To return null in a column, add the value None to the mapping with the key named after the column."
msgstr "För att returnera null i en kolumn så lägg till värdet None till mappningen med nyckelnamn taget från kolumnen."

#: plpy_typeio.c:1044
#, c-format
msgid "length of returned sequence did not match number of columns in row"
msgstr "längden på den returnerade sekvensen matchade inte antal kolumner i raden"

#: plpy_typeio.c:1155
#, c-format
msgid "attribute \"%s\" does not exist in Python object"
msgstr "attributet \"%s\" finns inte i Python-objektet"

#: plpy_typeio.c:1156
#, c-format
msgid "To return null in a column, let the returned object have an attribute named after column with value None."
msgstr "För att returnera null i en kolumn så låt det returnerade objektet ha ett attribut med namn efter kolumnen och med värdet None."

#: plpy_util.c:72
#, c-format
msgid "could not convert Python Unicode object to bytes"
msgstr "kunde inte konvertera Python-unicode-objekt till bytes"

#: plpy_util.c:78
#, c-format
msgid "could not extract bytes from encoded string"
msgstr "kunde inte extrahera bytes från kodad sträng"
